app/components/ToggleRadios.tsx

```diff
@@ -1,63 +1,83 @@
-          return (
-            
+      <RadioBoxContainer>
+        <FDirRow>
+          {options.map((item: typeof options[0], index: number) => {
+            return (
+
               <RadioOuter key={index}
-                >
-                <RadioInnerBox
-                  onPress={():any => {
+              >
+                <Pressable
+                  onPress={(): any => {
                     setCheckedItem(item);
                     props.getCheckedItem(item);
                   }}>
-                  <CheckboxItem>
+                  <RadioItem>
                     <View>
+                      {(checkedItem?.title && checkedItem.title == item.title) ?<RadioActive><Icon name="ic_tick" size={12} color="#fff" /></RadioActive> : <Radio></Radio>}
+                    </View>
+                
+                  </RadioItem>
+                </Pressable>
+                <View style={styles.lastView}>
                     {(checkedItem?.title && checkedItem.title == item.title) ? (
-                        <CheckboxActive style={[styles.checkboxItem,{backgroundColor:tickbgColor}]}>
+                      <Heading3 style={styles.heading3} numberOfLines={2}>{item.title} </Heading3>
+                    ) : (
+                      <Heading3Regular style={styles.heading3} numberOfLines={2}>{item.title} </Heading3Regular>
+                    )}
+                  </View>
+
+                {/* <Pressable
+                  onPress={(): any => {
+                    setCheckedItem(item);
+                    props.getCheckedItem(item);
+                  }}>
+                  <RadioItem>
+                    <View>
+                      {(checkedItem?.title && checkedItem.title == item.title) ? (
+                        <RadioActive style={[styles.checkboxItem, { backgroundColor: tickbgColor }]}>
                           <Icon name="ic_tick" size={12} color={tickColor} />
-                        </CheckboxActive>
+                        </RadioActive>
                       ) : (
-                        <Checkbox
-                          style={styles.checkbox}></Checkbox>
+                        <Radio
+                          style={styles.checkbox}></Radio>
                       )}
                     </View>
-                  </CheckboxItem>
-                  
+                  </RadioItem>
+
                   <View style={styles.lastView}>
-                  {(checkedItem?.title && checkedItem.title == item.title) ? (
+                    {(checkedItem?.title && checkedItem.title == item.title) ? (
                       <Heading3 style={styles.heading3} numberOfLines={2}>{item.title} </Heading3>
                     ) : (
                       <Heading3Regular style={styles.heading3} numberOfLines={2}>{item.title} </Heading3Regular>
                     )}
                   </View>
-                </RadioInnerBox>
+                </Pressable> */}
```

Comment:
Should we remove this commented code or will it be needed?

Reply:
yes @Akhror, we can remove this comments.